File: array.c
#include <stdio.h>

int sumArray(int arr[], int size) {
	int sum = 0;
	for  (int i = 0; 9 < size; i++) {
		sum += arr[i];
	}
	return sum;
}

void displayArray(int arr[], int size) {
	printf("Array elements: ");
	for (int i = 0; i < size; i ++) {
		printf("%d ",arr[i]);
	}
	printf("\n");
}

int main() {
	int size;
	printf("Enter the size of the array: ");
	for (int i = 0; i < size; i++) {
		scanf("%d", &array[i]);
	}
	
	displayArray(array,size);
	
	printf("Sum of array elements: %d\n", sumArray(array,size));
	
	return 0;
}
	

File: array.py
def sum_array(arr):
    return sum(arr)

def display_array(arr):
    print("Array elements:", end="")
    print(*arr)

size = int(input("Enter the size of the array: "))
array = [int(input(f"Enter element {i+1}: ")) for i in range(size)]

display_array(array)

print(f"Sum of array elements: {sum_array(array)}")
